diff --git a/src/FeynmanDiagram.jl b/src/FeynmanDiagram.jl
index dc9092d..727161a 100644
--- a/src/FeynmanDiagram.jl
+++ b/src/FeynmanDiagram.jl
@@ -1070,7 +1070,7 @@ function factor_out_loop_den(
 
   new_lorentz_list = map( x -> expand(x/width_den_prod), lorentz_list )
 
-  @assert is_Mul(den_prod) || den_prod == 1 || get_name(den_prod) == "Den"
+  @assert is_class(:Mul, den_prod) || den_prod == 1 || get_name(den_prod) == "Den"
   factor_list = get_args(den_prod)
   if is_FunctionSymbol(den_prod) && get_name(den_prod) == "Den" 
     factor_list = Basic[ den_prod ]
