diff --git a/Project.toml b/Project.toml
index 7351842..c1cd84d 100644
--- a/Project.toml
+++ b/Project.toml
@@ -13,6 +13,7 @@ OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Pipe = "b98c9c47-44ae-5843-9183-064241ee97a0"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+QGRAF_jll = "0e076a15-53bb-58c9-a2a4-288c9acedf6f"
 SymEngine = "123dc426-2d89-5057-bbad-38513e3affd8"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
diff --git a/src/FeAmGen.jl b/src/FeAmGen.jl
index d05da3a..783775f 100644
--- a/src/FeAmGen.jl
+++ b/src/FeAmGen.jl
@@ -32,6 +32,7 @@ using JLD2
 using Pipe
 
 using FORM_jll
+using QGRAF_jll
 
 export digest_seed_proc, generate_amp, generate_integral
 export generate_multi_yaml, generate_shiftUP_yaml
diff --git a/src/FeynmanDiagram.jl b/src/FeynmanDiagram.jl
index dc9092d..cf6432b 100644
--- a/src/FeynmanDiagram.jl
+++ b/src/FeynmanDiagram.jl
@@ -122,7 +122,8 @@ function generate_Feynman_diagram( model::Model, input::Dict{Any,Any} )
   if isfile( "qgraf_out.dat" ) == true
     rm( "qgraf_out.dat" )
   end # if
-  run( pipeline( `qgraf`, "qgraf.log" ) )
+  run( pipeline(qgraf(), "qgraf.log") )
+  # run( pipeline( `qgraf`, "qgraf.log" ) )
   @assert isfile( "qgraf_out.dat" )
 
   rm( "qgraf.dat" )
